app/views/disassociate_ducr_confirmation.scala.html

```diff
@@ -28,11 +28,14 @@
         headingText = messages("disassociateDucr.confirmation.heading", flash.get(FlashKeys.DUCR).getOrElse("-"))
     )
 
-    @components.paragraph(messages("movement.confirmation.statusInfo"), Some("status-info"))
+    @components.confirmation_status_info()
 
     <h2 id="what-next">@messages("movement.confirmation.whatNext")</h2>
 
-    @components.paragraph(messages(s"consolidation.confirmation.addOrShut"), Some("next-steps"))
+    <div id="next-steps">
+        <a href="@routes.ChoiceController.startSpecificJourney("associateDUCR")">@messages("consolidation.confirmation.addOrShut.part1")</a> @messages("consolidation.confirmation.addOrShut.part2")
```

Comment:
Those weird links will be replaced with better approach: https://stackoverflow.com/questions/28587760/html-parameter-passed-to-messages-gets-escaped-in-play-framework/28587899#28587899